apps/ops/models/job.py

```diff
@@ -181,13 +181,9 @@ def date_last_run(self):
     @property
     def summary(self):
         summary = {
-            "total": 0,
-            "success": 0,
+            "total": self.executions.count(),
+            "success": self.executions.filter(status=JobStatus.success).count(),
         }
-        for execution in self.executions.all():
-            summary["total"] += 1
-            if execution.is_success:
-                summary["success"] += 1
         return summary
 
     @property
```

Comment:
As an AI language model, I do not have access to any external code beyond what is mentioned here for context. However, based on the limited information provided:

**Code Inspection:** 

The given code appears correct according to the specifications provided (knowledge cutoff and current date). It should efficiently fetch the total number of executions ('executions' query filter 'status') and count successful ones ('executions'). The functions `date_last_run` and `summary` return the respective totals.

However, without running it directly with the given conditions or specific execution results (`self.executions.count()`), we cannot say with certainty it's optimal. 

**Suggestions:**

No optimizations suggested within scope - no known performance problems detected.

**Caveats (For Future Usage):**
This analysis only applies within these constraints and may need further checks during actual usage scenario.

If you provide additional context such as expected data, database schema, API endpoints etc., I would be able to optimize this further.